templates/CRM/Financial/Form/BatchTransaction.tpl

```diff
@@ -94,8 +94,9 @@ CRM.$(function($) {
     CRM.$('.crm-batch_transaction_search-accordion:not(.collapsed)').crmAccordionToggle();
   });
   var batchStatus = {/literal}{$statusID}{literal};
-  // build transaction listing only for open batches
-  if (batchStatus == 1) {
+  var validStatus = {/literal}{$validStatus}{literal};
+  // build transaction listing only for open/reopened batches
+  if (validStatus) {
```

Comment:
How about excluding the line set of line from 99-163 i.e 
{if $validStatus} include 99-162 {else} line 164 {/if}